src/api/v4/layer/layer.js

```diff
@@ -25,14 +36,18 @@ var metadataParser = require('./metadata/parser');
  * @fires carto.layer.styleChanged
  * @fires carto.layer.MetadataEvent
  * @example
- * // no options
+ * // Create a layer with no options
  * new carto.layer.Layer(citiesSource, citiesStyle);
  * @example
- * // with options
+ * // Create a layer indicating which columns in the data will throw events on mouse over
  * new carto.layer.Layer(citiesSource, citiesStyle, {
- *   featureClickColumns: [ 'name', 'population' ],
  *   featureOverColumns: [ 'name' ]
  * });
+ * @example
+ * // Listen to the event thrown when the mouse is over a feature
+ * layer.on(carto.layer.events.FEATURE_OVER, featureEvent => {
```

Comment:
Please use 'featureOver' here.

Reply:
you mean raw string? if so we got to change the examples too

Reply:
Yes, we're moving towards that in https://github.com/CartoDB/cartodb.js/pull/1963.